Build failure: python313Packages.pysdl2

### Nixpkgs version

- Unstable (25.11)

### Steps to reproduce

build `python313Packages.pysdl2`

### Can Hydra reproduce this build failure?

Yes, Hydra can reproduce this build failure.

### Link to Hydra build job

https://hydra.nixos.org/build/313877578

### Relevant log output

```console
Sourcing python-remove-tests-dir-hook
Sourcing python-catch-conflicts-hook.sh
Sourcing python-remove-bin-bytecode-hook.sh
Sourcing pypa-build-hook
Using pypaBuildPhase
Sourcing python-runtime-deps-check-hook
Using pythonRuntimeDepsCheckHook
Sourcing pypa-install-hook
Using pypaInstallPhase
Sourcing python-imports-check-hook.sh
Using pythonImportsCheckPhase
Sourcing python-namespaces-hook
Sourcing python-catch-conflicts-hook.sh
Sourcing pytest-check-hook
Using pytestCheckPhase
Running phase: unpackPhase
unpacking source archive /nix/store/2b67m4gq70hhyp679ka1f86ip5p6jsfi-source
source root is source
setting SOURCE_DATE_EPOCH to timestamp 315619200 of file "source/setup.py"
Running phase: patchPhase
applying patch /nix/store/ik97xvbdp6pfgpmim4vn1d9p9kyxzb9y-PySDL2-dll.patch
patching file sdl2/dll.py
patching file sdl2/sdlgfx.py
patching file sdl2/sdlimage.py
patching file sdl2/sdlmixer.py
patching file sdl2/sdlttf.py
Hunk #1 succeeded at 46 (offset 5 lines).
Running phase: updateAutotoolsGnuConfigScriptsPhase
Running phase: configurePhase
no configure script, doing nothing
Running phase: buildPhase
Executing pypaBuildPhase
Creating a wheel...
pypa build flags: --no-isolation --outdir dist/ --wheel
* Getting build dependencies for wheel...
/nix/store/jjqasz9apcfs8x1jm71dv81ayaqvfrln-python3.13-setuptools-80.9.0/lib/python3.13/site-packages/setuptools/dist.py:759: SetuptoolsDeprecationWarning: License classifiers are deprecated.
!!

        ********************************************************************************
        Please consider removing the following classifiers in favor of a SPDX license expression:

        License :: Public Domain
        License :: OSI Approved :: zlib/libpng License

        See https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#license for details.
        ********************************************************************************

!!
  self._finalize_license_expression()
running egg_info
creating PySDL2.egg-info
writing PySDL2.egg-info/PKG-INFO
writing dependency_links to PySDL2.egg-info/dependency_links.txt
writing top-level names to PySDL2.egg-info/top_level.txt
writing manifest file 'PySDL2.egg-info/SOURCES.txt'
reading manifest file 'PySDL2.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
warning: no directories found matching 'util'
warning: no files found matching '*.bat'
warning: no files found matching 'Makefile'
warning: no files found matching '*.rst'
no previously-included directories found matching 'doc/html'
no previously-included directories found matching 'doc/python.inv'
adding license file 'COPYING.txt'
adding license file 'AUTHORS.txt'
writing manifest file 'PySDL2.egg-info/SOURCES.txt'
* Building wheel...
/nix/store/jjqasz9apcfs8x1jm71dv81ayaqvfrln-python3.13-setuptools-80.9.0/lib/python3.13/site-packages/setuptools/dist.py:759: SetuptoolsDeprecationWarning: License classifiers are deprecated.
!!

        ********************************************************************************
        Please consider removing the following classifiers in favor of a SPDX license expression:

        License :: Public Domain
        License :: OSI Approved :: zlib/libpng License

        See https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#license for details.
        ********************************************************************************

!!
  self._finalize_license_expression()
running bdist_wheel
running build
running build_py
creating build/lib/sdl2
copying sdl2/sdlttf.py -> build/lib/sdl2
copying sdl2/sdlmixer.py -> build/lib/sdl2
copying sdl2/sdlimage.py -> build/lib/sdl2
copying sdl2/sdlgfx.py -> build/lib/sdl2
copying sdl2/dll.py -> build/lib/sdl2
copying sdl2/vulkan.py -> build/lib/sdl2
copying sdl2/video.py -> build/lib/sdl2
copying sdl2/version.py -> build/lib/sdl2
copying sdl2/touch.py -> build/lib/sdl2
copying sdl2/timer.py -> build/lib/sdl2
copying sdl2/syswm.py -> build/lib/sdl2
copying sdl2/surface.py -> build/lib/sdl2
copying sdl2/stdinc.py -> build/lib/sdl2
copying sdl2/shape.py -> build/lib/sdl2
copying sdl2/sensor.py -> build/lib/sdl2
copying sdl2/scancode.py -> build/lib/sdl2
copying sdl2/rwops.py -> build/lib/sdl2
copying sdl2/render.py -> build/lib/sdl2
copying sdl2/rect.py -> build/lib/sdl2
copying sdl2/power.py -> build/lib/sdl2
copying sdl2/platform.py -> build/lib/sdl2
copying sdl2/pixels.py -> build/lib/sdl2
copying sdl2/mouse.py -> build/lib/sdl2
copying sdl2/misc.py -> build/lib/sdl2
copying sdl2/metal.py -> build/lib/sdl2
copying sdl2/messagebox.py -> build/lib/sdl2
copying sdl2/log.py -> build/lib/sdl2
copying sdl2/locale.py -> build/lib/sdl2
copying sdl2/loadso.py -> build/lib/sdl2
copying sdl2/keycode.py -> build/lib/sdl2
copying sdl2/keyboard.py -> build/lib/sdl2
copying sdl2/joystick.py -> build/lib/sdl2
copying sdl2/hints.py -> build/lib/sdl2
copying sdl2/hidapi.py -> build/lib/sdl2
copying sdl2/haptic.py -> build/lib/sdl2
copying sdl2/guid.py -> build/lib/sdl2
copying sdl2/gesture.py -> build/lib/sdl2
copying sdl2/gamecontroller.py -> build/lib/sdl2
copying sdl2/filesystem.py -> build/lib/sdl2
copying sdl2/events.py -> build/lib/sdl2
copying sdl2/error.py -> build/lib/sdl2
copying sdl2/endian.py -> build/lib/sdl2
copying sdl2/cpuinfo.py -> build/lib/sdl2
copying sdl2/clipboard.py -> build/lib/sdl2
copying sdl2/blendmode.py -> build/lib/sdl2
copying sdl2/audio.py -> build/lib/sdl2
copying sdl2/_sdl_init.py -> build/lib/sdl2
copying sdl2/_internal.py -> build/lib/sdl2
copying sdl2/__init__.py -> build/lib/sdl2
creating build/lib/sdl2/ext
copying sdl2/ext/window.py -> build/lib/sdl2/ext
copying sdl2/ext/uisystem.py -> build/lib/sdl2/ext
copying sdl2/ext/ttf.py -> build/lib/sdl2/ext
copying sdl2/ext/surface.py -> build/lib/sdl2/ext
copying sdl2/ext/spritesystem.py -> build/lib/sdl2/ext
copying sdl2/ext/sprite.py -> build/lib/sdl2/ext
copying sdl2/ext/resources.py -> build/lib/sdl2/ext
copying sdl2/ext/renderer.py -> build/lib/sdl2/ext
copying sdl2/ext/pixelaccess.py -> build/lib/sdl2/ext
copying sdl2/ext/particles.py -> build/lib/sdl2/ext
copying sdl2/ext/msgbox.py -> build/lib/sdl2/ext
copying sdl2/ext/mouse.py -> build/lib/sdl2/ext
copying sdl2/ext/input.py -> build/lib/sdl2/ext
copying sdl2/ext/image.py -> build/lib/sdl2/ext
copying sdl2/ext/events.py -> build/lib/sdl2/ext
copying sdl2/ext/err.py -> build/lib/sdl2/ext
copying sdl2/ext/ebs.py -> build/lib/sdl2/ext
copying sdl2/ext/draw.py -> build/lib/sdl2/ext
copying sdl2/ext/displays.py -> build/lib/sdl2/ext
copying sdl2/ext/compat.py -> build/lib/sdl2/ext
copying sdl2/ext/common.py -> build/lib/sdl2/ext
copying sdl2/ext/colorpalettes.py -> build/lib/sdl2/ext
copying sdl2/ext/color.py -> build/lib/sdl2/ext
copying sdl2/ext/bitmapfont.py -> build/lib/sdl2/ext
copying sdl2/ext/array.py -> build/lib/sdl2/ext
copying sdl2/ext/algorithms.py -> build/lib/sdl2/ext
copying sdl2/ext/__init__.py -> build/lib/sdl2/ext
creating build/lib/sdl2/test
copying sdl2/test/vulkan_test.py -> build/lib/sdl2/test
copying sdl2/test/video_test.py -> build/lib/sdl2/test
copying sdl2/test/version_test.py -> build/lib/sdl2/test
copying sdl2/test/touch_test.py -> build/lib/sdl2/test
copying sdl2/test/timer_test.py -> build/lib/sdl2/test
copying sdl2/test/syswm_test.py -> build/lib/sdl2/test
copying sdl2/test/surface_test.py -> build/lib/sdl2/test
copying sdl2/test/shape_test.py -> build/lib/sdl2/test
copying sdl2/test/sensor_test.py -> build/lib/sdl2/test
copying sdl2/test/sdlttf_test.py -> build/lib/sdl2/test
copying sdl2/test/sdlmixer_test.py -> build/lib/sdl2/test
copying sdl2/test/sdlimage_test.py -> build/lib/sdl2/test
copying sdl2/test/sdlgfx_test.py -> build/lib/sdl2/test
copying sdl2/test/sdl_test.py -> build/lib/sdl2/test
copying sdl2/test/sdl2ext_window_test.py -> build/lib/sdl2/test
copying sdl2/test/sdl2ext_uisystem_test.py -> build/lib/sdl2/test
copying sdl2/test/sdl2ext_test.py -> build/lib/sdl2/test
copying sdl2/test/sdl2ext_surface_test.py -> build/lib/sdl2/test
copying sdl2/test/sdl2ext_spritesystem_test.py -> build/lib/sdl2/test
copying sdl2/test/sdl2ext_sprite_test.py -> build/lib/sdl2/test
copying sdl2/test/sdl2ext_resources_test.py -> build/lib/sdl2/test
copying sdl2/test/sdl2ext_renderer_test.py -> build/lib/sdl2/test
copying sdl2/test/sdl2ext_pixelaccess_test.py -> build/lib/sdl2/test
copying sdl2/test/sdl2ext_particles_test.py -> build/lib/sdl2/test
copying sdl2/test/sdl2ext_msgbox_test.py -> build/lib/sdl2/test
copying sdl2/test/sdl2ext_mouse_test.py -> build/lib/sdl2/test
copying sdl2/test/sdl2ext_input_test.py -> build/lib/sdl2/test
copying sdl2/test/sdl2ext_image_test.py -> build/lib/sdl2/test
copying sdl2/test/sdl2ext_font_test.py -> build/lib/sdl2/test
copying sdl2/test/sdl2ext_events_test.py -> build/lib/sdl2/test
copying sdl2/test/sdl2ext_ebs_test.py -> build/lib/sdl2/test
copying sdl2/test/sdl2ext_draw_test.py -> build/lib/sdl2/test
copying sdl2/test/sdl2ext_displays_test.py -> build/lib/sdl2/test
copying sdl2/test/sdl2ext_color_test.py -> build/lib/sdl2/test
copying sdl2/test/sdl2ext_array_test.py -> build/lib/sdl2/test
copying sdl2/test/sdl2ext_algorithms_test.py -> build/lib/sdl2/test
copying sdl2/test/rwops_test.py -> build/lib/sdl2/test
copying sdl2/test/render_test.py -> build/lib/sdl2/test
copying sdl2/test/rect_test.py -> build/lib/sdl2/test
copying sdl2/test/power_test.py -> build/lib/sdl2/test
copying sdl2/test/platform_test.py -> build/lib/sdl2/test
copying sdl2/test/pixels_test.py -> build/lib/sdl2/test
copying sdl2/test/mouse_test.py -> build/lib/sdl2/test
copying sdl2/test/metal_test.py -> build/lib/sdl2/test
copying sdl2/test/log_test.py -> build/lib/sdl2/test
copying sdl2/test/locale_test.py -> build/lib/sdl2/test
copying sdl2/test/keyboard_test.py -> build/lib/sdl2/test
copying sdl2/test/joystick_test.py -> build/lib/sdl2/test
copying sdl2/test/hints_test.py -> build/lib/sdl2/test
copying sdl2/test/hidapi_test.py -> build/lib/sdl2/test
copying sdl2/test/guid_test.py -> build/lib/sdl2/test
copying sdl2/test/gamecontroller_test.py -> build/lib/sdl2/test
copying sdl2/test/filesystem_test.py -> build/lib/sdl2/test
copying sdl2/test/events_test.py -> build/lib/sdl2/test
copying sdl2/test/error_test.py -> build/lib/sdl2/test
copying sdl2/test/endian_test.py -> build/lib/sdl2/test
copying sdl2/test/cpuinfo_test.py -> build/lib/sdl2/test
copying sdl2/test/conftest.py -> build/lib/sdl2/test
copying sdl2/test/clipboard_test.py -> build/lib/sdl2/test
copying sdl2/test/blendmode_test.py -> build/lib/sdl2/test
copying sdl2/test/audio_test.py -> build/lib/sdl2/test
copying sdl2/test/__init__.py -> build/lib/sdl2/test
creating build/lib/sdl2/examples
copying examples/ttf.py -> build/lib/sdl2/examples
copying examples/transfomations.py -> build/lib/sdl2/examples
copying examples/sdl2hello.py -> build/lib/sdl2/examples
copying examples/pong.py -> build/lib/sdl2/examples
copying examples/pixelaccess.py -> build/lib/sdl2/examples
copying examples/particles.py -> build/lib/sdl2/examples
copying examples/opengl.py -> build/lib/sdl2/examples
copying examples/helloworld.py -> build/lib/sdl2/examples
copying examples/gui.py -> build/lib/sdl2/examples
copying examples/gfxdrawing.py -> build/lib/sdl2/examples
copying examples/draw.py -> build/lib/sdl2/examples
copying examples/colorpalettes.py -> build/lib/sdl2/examples
copying examples/__init__.py -> build/lib/sdl2/examples
creating build/lib/sdl2/test/resources
copying sdl2/test/resources/tuffy.ttf -> build/lib/sdl2/test/resources
copying sdl2/test/resources/surfacetest.xpm -> build/lib/sdl2/test/resources
copying sdl2/test/resources/surfacetest.xcf -> build/lib/sdl2/test/resources
copying sdl2/test/resources/surfacetest.webp -> build/lib/sdl2/test/resources
copying sdl2/test/resources/surfacetest.tif -> build/lib/sdl2/test/resources
copying sdl2/test/resources/surfacetest.tga -> build/lib/sdl2/test/resources
copying sdl2/test/resources/surfacetest.svg -> build/lib/sdl2/test/resources
copying sdl2/test/resources/surfacetest.qoi -> build/lib/sdl2/test/resources
copying sdl2/test/resources/surfacetest.ppm -> build/lib/sdl2/test/resources
copying sdl2/test/resources/surfacetest.pnm -> build/lib/sdl2/test/resources
copying sdl2/test/resources/surfacetest.png -> build/lib/sdl2/test/resources
copying sdl2/test/resources/surfacetest.pgm -> build/lib/sdl2/test/resources
copying sdl2/test/resources/surfacetest.pcx -> build/lib/sdl2/test/resources
copying sdl2/test/resources/surfacetest.pbm -> build/lib/sdl2/test/resources
copying sdl2/test/resources/surfacetest.lbm -> build/lib/sdl2/test/resources
copying sdl2/test/resources/surfacetest.jpg -> build/lib/sdl2/test/resources
copying sdl2/test/resources/surfacetest.ico -> build/lib/sdl2/test/resources
copying sdl2/test/resources/surfacetest.gif -> build/lib/sdl2/test/resources
copying sdl2/test/resources/surfacetest.cur -> build/lib/sdl2/test/resources
copying sdl2/test/resources/surfacetest.bmp -> build/lib/sdl2/test/resources
copying sdl2/test/resources/soundtest.mp3 -> build/lib/sdl2/test/resources
copying sdl2/test/resources/rwopstest.txt -> build/lib/sdl2/test/resources
copying sdl2/test/resources/resources.zip -> build/lib/sdl2/test/resources
copying sdl2/test/resources/resources.tar.gz -> build/lib/sdl2/test/resources
copying sdl2/test/resources/font.bmp -> build/lib/sdl2/test/resources
copying sdl2/test/resources/animationtest.webp -> build/lib/sdl2/test/resources
copying sdl2/test/resources/animationtest.gif -> build/lib/sdl2/test/resources
creating build/lib/sdl2/examples/resources
copying examples/resources/tuffy.ttf -> build/lib/sdl2/examples/resources
copying examples/resources/textentry.bmp -> build/lib/sdl2/examples/resources
copying examples/resources/testimage.svg -> build/lib/sdl2/examples/resources
copying examples/resources/star.png -> build/lib/sdl2/examples/resources
copying examples/resources/square.png -> build/lib/sdl2/examples/resources
copying examples/resources/hello.bmp -> build/lib/sdl2/examples/resources
copying examples/resources/font.bmp -> build/lib/sdl2/examples/resources
copying examples/resources/circle.png -> build/lib/sdl2/examples/resources
copying examples/resources/button.bmp -> build/lib/sdl2/examples/resources
installing to build/bdist.linux-x86_64/wheel
running install
running install_lib
creating build/bdist.linux-x86_64/wheel
creating build/bdist.linux-x86_64/wheel/sdl2
creating build/bdist.linux-x86_64/wheel/sdl2/examples
creating build/bdist.linux-x86_64/wheel/sdl2/examples/resources
copying build/lib/sdl2/examples/resources/button.bmp -> build/bdist.linux-x86_64/wheel/./sdl2/examples/resources
copying build/lib/sdl2/examples/resources/circle.png -> build/bdist.linux-x86_64/wheel/./sdl2/examples/resources
copying build/lib/sdl2/examples/resources/font.bmp -> build/bdist.linux-x86_64/wheel/./sdl2/examples/resources
copying build/lib/sdl2/examples/resources/hello.bmp -> build/bdist.linux-x86_64/wheel/./sdl2/examples/resources
copying build/lib/sdl2/examples/resources/square.png -> build/bdist.linux-x86_64/wheel/./sdl2/examples/resources
copying build/lib/sdl2/examples/resources/star.png -> build/bdist.linux-x86_64/wheel/./sdl2/examples/resources
copying build/lib/sdl2/examples/resources/testimage.svg -> build/bdist.linux-x86_64/wheel/./sdl2/examples/resources
copying build/lib/sdl2/examples/resources/textentry.bmp -> build/bdist.linux-x86_64/wheel/./sdl2/examples/resources
copying build/lib/sdl2/examples/resources/tuffy.ttf -> build/bdist.linux-x86_64/wheel/./sdl2/examples/resources
copying build/lib/sdl2/examples/__init__.py -> build/bdist.linux-x86_64/wheel/./sdl2/examples
copying build/lib/sdl2/examples/colorpalettes.py -> build/bdist.linux-x86_64/wheel/./sdl2/examples
copying build/lib/sdl2/examples/draw.py -> build/bdist.linux-x86_64/wheel/./sdl2/examples
copying build/lib/sdl2/examples/gfxdrawing.py -> build/bdist.linux-x86_64/wheel/./sdl2/examples
copying build/lib/sdl2/examples/gui.py -> build/bdist.linux-x86_64/wheel/./sdl2/examples
copying build/lib/sdl2/examples/helloworld.py -> build/bdist.linux-x86_64/wheel/./sdl2/examples
copying build/lib/sdl2/examples/opengl.py -> build/bdist.linux-x86_64/wheel/./sdl2/examples
copying build/lib/sdl2/examples/particles.py -> build/bdist.linux-x86_64/wheel/./sdl2/examples
copying build/lib/sdl2/examples/pixelaccess.py -> build/bdist.linux-x86_64/wheel/./sdl2/examples
copying build/lib/sdl2/examples/pong.py -> build/bdist.linux-x86_64/wheel/./sdl2/examples
copying build/lib/sdl2/examples/sdl2hello.py -> build/bdist.linux-x86_64/wheel/./sdl2/examples
copying build/lib/sdl2/examples/transfomations.py -> build/bdist.linux-x86_64/wheel/./sdl2/examples
copying build/lib/sdl2/examples/ttf.py -> build/bdist.linux-x86_64/wheel/./sdl2/examples
creating build/bdist.linux-x86_64/wheel/sdl2/test
creating build/bdist.linux-x86_64/wheel/sdl2/test/resources
copying build/lib/sdl2/test/resources/animationtest.gif -> build/bdist.linux-x86_64/wheel/./sdl2/test/resources
copying build/lib/sdl2/test/resources/animationtest.webp -> build/bdist.linux-x86_64/wheel/./sdl2/test/resources
copying build/lib/sdl2/test/resources/font.bmp -> build/bdist.linux-x86_64/wheel/./sdl2/test/resources
copying build/lib/sdl2/test/resources/resources.tar.gz -> build/bdist.linux-x86_64/wheel/./sdl2/test/resources
copying build/lib/sdl2/test/resources/resources.zip -> build/bdist.linux-x86_64/wheel/./sdl2/test/resources
copying build/lib/sdl2/test/resources/rwopstest.txt -> build/bdist.linux-x86_64/wheel/./sdl2/test/resources
copying build/lib/sdl2/test/resources/soundtest.mp3 -> build/bdist.linux-x86_64/wheel/./sdl2/test/resources
copying build/lib/sdl2/test/resources/surfacetest.bmp -> build/bdist.linux-x86_64/wheel/./sdl2/test/resources
copying build/lib/sdl2/test/resources/surfacetest.cur -> build/bdist.linux-x86_64/wheel/./sdl2/test/resources
copying build/lib/sdl2/test/resources/surfacetest.gif -> build/bdist.linux-x86_64/wheel/./sdl2/test/resources
copying build/lib/sdl2/test/resources/surfacetest.ico -> build/bdist.linux-x86_64/wheel/./sdl2/test/resources
copying build/lib/sdl2/test/resources/surfacetest.jpg -> build/bdist.linux-x86_64/wheel/./sdl2/test/resources
copying build/lib/sdl2/test/resources/surfacetest.lbm -> build/bdist.linux-x86_64/wheel/./sdl2/test/resources
copying build/lib/sdl2/test/resources/surfacetest.pbm -> build/bdist.linux-x86_64/wheel/./sdl2/test/resources
copying build/lib/sdl2/test/resources/surfacetest.pcx -> build/bdist.linux-x86_64/wheel/./sdl2/test/resources
copying build/lib/sdl2/test/resources/surfacetest.pgm -> build/bdist.linux-x86_64/wheel/./sdl2/test/resources
copying build/lib/sdl2/test/resources/surfacetest.png -> build/bdist.linux-x86_64/wheel/./sdl2/test/resources
copying build/lib/sdl2/test/resources/surfacetest.pnm -> build/bdist.linux-x86_64/wheel/./sdl2/test/resources
copying build/lib/sdl2/test/resources/surfacetest.ppm -> build/bdist.linux-x86_64/wheel/./sdl2/test/resources
copying build/lib/sdl2/test/resources/surfacetest.qoi -> build/bdist.linux-x86_64/wheel/./sdl2/test/resources
copying build/lib/sdl2/test/resources/surfacetest.svg -> build/bdist.linux-x86_64/wheel/./sdl2/test/resources
copying build/lib/sdl2/test/resources/surfacetest.tga -> build/bdist.linux-x86_64/wheel/./sdl2/test/resources
copying build/lib/sdl2/test/resources/surfacetest.tif -> build/bdist.linux-x86_64/wheel/./sdl2/test/resources
copying build/lib/sdl2/test/resources/surfacetest.webp -> build/bdist.linux-x86_64/wheel/./sdl2/test/resources
copying build/lib/sdl2/test/resources/surfacetest.xcf -> build/bdist.linux-x86_64/wheel/./sdl2/test/resources
copying build/lib/sdl2/test/resources/surfacetest.xpm -> build/bdist.linux-x86_64/wheel/./sdl2/test/resources
copying build/lib/sdl2/test/resources/tuffy.ttf -> build/bdist.linux-x86_64/wheel/./sdl2/test/resources
copying build/lib/sdl2/test/__init__.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/audio_test.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/blendmode_test.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/clipboard_test.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/conftest.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/cpuinfo_test.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/endian_test.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/error_test.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/events_test.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/filesystem_test.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/gamecontroller_test.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/guid_test.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/hidapi_test.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/hints_test.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/joystick_test.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/keyboard_test.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/locale_test.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/log_test.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/metal_test.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/mouse_test.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/pixels_test.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/platform_test.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/power_test.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/rect_test.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/render_test.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/rwops_test.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/sdl2ext_algorithms_test.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/sdl2ext_array_test.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/sdl2ext_color_test.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/sdl2ext_displays_test.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/sdl2ext_draw_test.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/sdl2ext_ebs_test.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/sdl2ext_events_test.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/sdl2ext_font_test.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/sdl2ext_image_test.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/sdl2ext_input_test.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/sdl2ext_mouse_test.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/sdl2ext_msgbox_test.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/sdl2ext_particles_test.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/sdl2ext_pixelaccess_test.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/sdl2ext_renderer_test.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/sdl2ext_resources_test.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/sdl2ext_sprite_test.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/sdl2ext_spritesystem_test.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/sdl2ext_surface_test.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/sdl2ext_test.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/sdl2ext_uisystem_test.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/sdl2ext_window_test.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/sdl_test.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/sdlgfx_test.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/sdlimage_test.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/sdlmixer_test.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/sdlttf_test.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/sensor_test.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/shape_test.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/surface_test.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/syswm_test.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/timer_test.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/touch_test.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/version_test.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/video_test.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
copying build/lib/sdl2/test/vulkan_test.py -> build/bdist.linux-x86_64/wheel/./sdl2/test
creating build/bdist.linux-x86_64/wheel/sdl2/ext
copying build/lib/sdl2/ext/__init__.py -> build/bdist.linux-x86_64/wheel/./sdl2/ext
copying build/lib/sdl2/ext/algorithms.py -> build/bdist.linux-x86_64/wheel/./sdl2/ext
copying build/lib/sdl2/ext/array.py -> build/bdist.linux-x86_64/wheel/./sdl2/ext
copying build/lib/sdl2/ext/bitmapfont.py -> build/bdist.linux-x86_64/wheel/./sdl2/ext
copying build/lib/sdl2/ext/color.py -> build/bdist.linux-x86_64/wheel/./sdl2/ext
copying build/lib/sdl2/ext/colorpalettes.py -> build/bdist.linux-x86_64/wheel/./sdl2/ext
copying build/lib/sdl2/ext/common.py -> build/bdist.linux-x86_64/wheel/./sdl2/ext
copying build/lib/sdl2/ext/compat.py -> build/bdist.linux-x86_64/wheel/./sdl2/ext
copying build/lib/sdl2/ext/displays.py -> build/bdist.linux-x86_64/wheel/./sdl2/ext
copying build/lib/sdl2/ext/draw.py -> build/bdist.linux-x86_64/wheel/./sdl2/ext
copying build/lib/sdl2/ext/ebs.py -> build/bdist.linux-x86_64/wheel/./sdl2/ext
copying build/lib/sdl2/ext/err.py -> build/bdist.linux-x86_64/wheel/./sdl2/ext
copying build/lib/sdl2/ext/events.py -> build/bdist.linux-x86_64/wheel/./sdl2/ext
copying build/lib/sdl2/ext/image.py -> build/bdist.linux-x86_64/wheel/./sdl2/ext
copying build/lib/sdl2/ext/input.py -> build/bdist.linux-x86_64/wheel/./sdl2/ext
copying build/lib/sdl2/ext/mouse.py -> build/bdist.linux-x86_64/wheel/./sdl2/ext
copying build/lib/sdl2/ext/msgbox.py -> build/bdist.linux-x86_64/wheel/./sdl2/ext
copying build/lib/sdl2/ext/particles.py -> build/bdist.linux-x86_64/wheel/./sdl2/ext
copying build/lib/sdl2/ext/pixelaccess.py -> build/bdist.linux-x86_64/wheel/./sdl2/ext
copying build/lib/sdl2/ext/renderer.py -> build/bdist.linux-x86_64/wheel/./sdl2/ext
copying build/lib/sdl2/ext/resources.py -> build/bdist.linux-x86_64/wheel/./sdl2/ext
copying build/lib/sdl2/ext/sprite.py -> build/bdist.linux-x86_64/wheel/./sdl2/ext
copying build/lib/sdl2/ext/spritesystem.py -> build/bdist.linux-x86_64/wheel/./sdl2/ext
copying build/lib/sdl2/ext/surface.py -> build/bdist.linux-x86_64/wheel/./sdl2/ext
copying build/lib/sdl2/ext/ttf.py -> build/bdist.linux-x86_64/wheel/./sdl2/ext
copying build/lib/sdl2/ext/uisystem.py -> build/bdist.linux-x86_64/wheel/./sdl2/ext
copying build/lib/sdl2/ext/window.py -> build/bdist.linux-x86_64/wheel/./sdl2/ext
copying build/lib/sdl2/__init__.py -> build/bdist.linux-x86_64/wheel/./sdl2
copying build/lib/sdl2/_internal.py -> build/bdist.linux-x86_64/wheel/./sdl2
copying build/lib/sdl2/_sdl_init.py -> build/bdist.linux-x86_64/wheel/./sdl2
copying build/lib/sdl2/audio.py -> build/bdist.linux-x86_64/wheel/./sdl2
copying build/lib/sdl2/blendmode.py -> build/bdist.linux-x86_64/wheel/./sdl2
copying build/lib/sdl2/clipboard.py -> build/bdist.linux-x86_64/wheel/./sdl2
copying build/lib/sdl2/cpuinfo.py -> build/bdist.linux-x86_64/wheel/./sdl2
copying build/lib/sdl2/endian.py -> build/bdist.linux-x86_64/wheel/./sdl2
copying build/lib/sdl2/error.py -> build/bdist.linux-x86_64/wheel/./sdl2
copying build/lib/sdl2/events.py -> build/bdist.linux-x86_64/wheel/./sdl2
copying build/lib/sdl2/filesystem.py -> build/bdist.linux-x86_64/wheel/./sdl2
copying build/lib/sdl2/gamecontroller.py -> build/bdist.linux-x86_64/wheel/./sdl2
copying build/lib/sdl2/gesture.py -> build/bdist.linux-x86_64/wheel/./sdl2
copying build/lib/sdl2/guid.py -> build/bdist.linux-x86_64/wheel/./sdl2
copying build/lib/sdl2/haptic.py -> build/bdist.linux-x86_64/wheel/./sdl2
copying build/lib/sdl2/hidapi.py -> build/bdist.linux-x86_64/wheel/./sdl2
copying build/lib/sdl2/hints.py -> build/bdist.linux-x86_64/wheel/./sdl2
copying build/lib/sdl2/joystick.py -> build/bdist.linux-x86_64/wheel/./sdl2
copying build/lib/sdl2/keyboard.py -> build/bdist.linux-x86_64/wheel/./sdl2
copying build/lib/sdl2/keycode.py -> build/bdist.linux-x86_64/wheel/./sdl2
copying build/lib/sdl2/loadso.py -> build/bdist.linux-x86_64/wheel/./sdl2
copying build/lib/sdl2/locale.py -> build/bdist.linux-x86_64/wheel/./sdl2
copying build/lib/sdl2/log.py -> build/bdist.linux-x86_64/wheel/./sdl2
copying build/lib/sdl2/messagebox.py -> build/bdist.linux-x86_64/wheel/./sdl2
copying build/lib/sdl2/metal.py -> build/bdist.linux-x86_64/wheel/./sdl2
copying build/lib/sdl2/misc.py -> build/bdist.linux-x86_64/wheel/./sdl2
copying build/lib/sdl2/mouse.py -> build/bdist.linux-x86_64/wheel/./sdl2
copying build/lib/sdl2/pixels.py -> build/bdist.linux-x86_64/wheel/./sdl2
copying build/lib/sdl2/platform.py -> build/bdist.linux-x86_64/wheel/./sdl2
copying build/lib/sdl2/power.py -> build/bdist.linux-x86_64/wheel/./sdl2
copying build/lib/sdl2/rect.py -> build/bdist.linux-x86_64/wheel/./sdl2
copying build/lib/sdl2/render.py -> build/bdist.linux-x86_64/wheel/./sdl2
copying build/lib/sdl2/rwops.py -> build/bdist.linux-x86_64/wheel/./sdl2
copying build/lib/sdl2/scancode.py -> build/bdist.linux-x86_64/wheel/./sdl2
copying build/lib/sdl2/sensor.py -> build/bdist.linux-x86_64/wheel/./sdl2
copying build/lib/sdl2/shape.py -> build/bdist.linux-x86_64/wheel/./sdl2
copying build/lib/sdl2/stdinc.py -> build/bdist.linux-x86_64/wheel/./sdl2
copying build/lib/sdl2/surface.py -> build/bdist.linux-x86_64/wheel/./sdl2
copying build/lib/sdl2/syswm.py -> build/bdist.linux-x86_64/wheel/./sdl2
copying build/lib/sdl2/timer.py -> build/bdist.linux-x86_64/wheel/./sdl2
copying build/lib/sdl2/touch.py -> build/bdist.linux-x86_64/wheel/./sdl2
copying build/lib/sdl2/version.py -> build/bdist.linux-x86_64/wheel/./sdl2
copying build/lib/sdl2/video.py -> build/bdist.linux-x86_64/wheel/./sdl2
copying build/lib/sdl2/vulkan.py -> build/bdist.linux-x86_64/wheel/./sdl2
copying build/lib/sdl2/dll.py -> build/bdist.linux-x86_64/wheel/./sdl2
copying build/lib/sdl2/sdlgfx.py -> build/bdist.linux-x86_64/wheel/./sdl2
copying build/lib/sdl2/sdlimage.py -> build/bdist.linux-x86_64/wheel/./sdl2
copying build/lib/sdl2/sdlmixer.py -> build/bdist.linux-x86_64/wheel/./sdl2
copying build/lib/sdl2/sdlttf.py -> build/bdist.linux-x86_64/wheel/./sdl2
running install_egg_info
running egg_info
writing PySDL2.egg-info/PKG-INFO
writing dependency_links to PySDL2.egg-info/dependency_links.txt
writing top-level names to PySDL2.egg-info/top_level.txt
reading manifest file 'PySDL2.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
warning: no directories found matching 'util'
warning: no files found matching '*.bat'
warning: no files found matching 'Makefile'
warning: no files found matching '*.rst'
no previously-included directories found matching 'doc/html'
no previously-included directories found matching 'doc/python.inv'
adding license file 'COPYING.txt'
adding license file 'AUTHORS.txt'
writing manifest file 'PySDL2.egg-info/SOURCES.txt'
Copying PySDL2.egg-info to build/bdist.linux-x86_64/wheel/./PySDL2-0.9.18a1-py3.13.egg-info
running install_scripts
creating build/bdist.linux-x86_64/wheel/pysdl2-0.9.18a1.dist-info/WHEEL
creating '/build/source/dist/.tmp-qgqqozzl/pysdl2-0.9.18a1-py3-none-any.whl' and adding 'build/bdist.linux-x86_64/wheel' to it
adding 'pysdl2-0.9.18a1.dist-info/licenses/AUTHORS.txt'
adding 'pysdl2-0.9.18a1.dist-info/licenses/COPYING.txt'
adding 'sdl2/__init__.py'
adding 'sdl2/_internal.py'
adding 'sdl2/_sdl_init.py'
adding 'sdl2/audio.py'
adding 'sdl2/blendmode.py'
adding 'sdl2/clipboard.py'
adding 'sdl2/cpuinfo.py'
adding 'sdl2/dll.py'
adding 'sdl2/endian.py'
adding 'sdl2/error.py'
adding 'sdl2/events.py'
adding 'sdl2/filesystem.py'
adding 'sdl2/gamecontroller.py'
adding 'sdl2/gesture.py'
adding 'sdl2/guid.py'
adding 'sdl2/haptic.py'
adding 'sdl2/hidapi.py'
adding 'sdl2/hints.py'
adding 'sdl2/joystick.py'
adding 'sdl2/keyboard.py'
adding 'sdl2/keycode.py'
adding 'sdl2/loadso.py'
adding 'sdl2/locale.py'
adding 'sdl2/log.py'
adding 'sdl2/messagebox.py'
adding 'sdl2/metal.py'
adding 'sdl2/misc.py'
adding 'sdl2/mouse.py'
adding 'sdl2/pixels.py'
adding 'sdl2/platform.py'
adding 'sdl2/power.py'
adding 'sdl2/rect.py'
adding 'sdl2/render.py'
adding 'sdl2/rwops.py'
adding 'sdl2/scancode.py'
adding 'sdl2/sdlgfx.py'
adding 'sdl2/sdlimage.py'
adding 'sdl2/sdlmixer.py'
adding 'sdl2/sdlttf.py'
adding 'sdl2/sensor.py'
adding 'sdl2/shape.py'
adding 'sdl2/stdinc.py'
adding 'sdl2/surface.py'
adding 'sdl2/syswm.py'
adding 'sdl2/timer.py'
adding 'sdl2/touch.py'
adding 'sdl2/version.py'
adding 'sdl2/video.py'
adding 'sdl2/vulkan.py'
adding 'sdl2/examples/__init__.py'
adding 'sdl2/examples/colorpalettes.py'
adding 'sdl2/examples/draw.py'
adding 'sdl2/examples/gfxdrawing.py'
adding 'sdl2/examples/gui.py'
adding 'sdl2/examples/helloworld.py'
adding 'sdl2/examples/opengl.py'
adding 'sdl2/examples/particles.py'
adding 'sdl2/examples/pixelaccess.py'
adding 'sdl2/examples/pong.py'
adding 'sdl2/examples/sdl2hello.py'
adding 'sdl2/examples/transfomations.py'
adding 'sdl2/examples/ttf.py'
adding 'sdl2/examples/resources/button.bmp'
adding 'sdl2/examples/resources/circle.png'
adding 'sdl2/examples/resources/font.bmp'
adding 'sdl2/examples/resources/hello.bmp'
adding 'sdl2/examples/resources/square.png'
adding 'sdl2/examples/resources/star.png'
adding 'sdl2/examples/resources/testimage.svg'
adding 'sdl2/examples/resources/textentry.bmp'
adding 'sdl2/examples/resources/tuffy.ttf'
adding 'sdl2/ext/__init__.py'
adding 'sdl2/ext/algorithms.py'
adding 'sdl2/ext/array.py'
adding 'sdl2/ext/bitmapfont.py'
adding 'sdl2/ext/color.py'
adding 'sdl2/ext/colorpalettes.py'
adding 'sdl2/ext/common.py'
adding 'sdl2/ext/compat.py'
adding 'sdl2/ext/displays.py'
adding 'sdl2/ext/draw.py'
adding 'sdl2/ext/ebs.py'
adding 'sdl2/ext/err.py'
adding 'sdl2/ext/events.py'
adding 'sdl2/ext/image.py'
adding 'sdl2/ext/input.py'
adding 'sdl2/ext/mouse.py'
adding 'sdl2/ext/msgbox.py'
adding 'sdl2/ext/particles.py'
adding 'sdl2/ext/pixelaccess.py'
adding 'sdl2/ext/renderer.py'
adding 'sdl2/ext/resources.py'
adding 'sdl2/ext/sprite.py'
adding 'sdl2/ext/spritesystem.py'
adding 'sdl2/ext/surface.py'
adding 'sdl2/ext/ttf.py'
adding 'sdl2/ext/uisystem.py'
adding 'sdl2/ext/window.py'
adding 'sdl2/test/__init__.py'
adding 'sdl2/test/audio_test.py'
adding 'sdl2/test/blendmode_test.py'
adding 'sdl2/test/clipboard_test.py'
adding 'sdl2/test/conftest.py'
adding 'sdl2/test/cpuinfo_test.py'
adding 'sdl2/test/endian_test.py'
adding 'sdl2/test/error_test.py'
adding 'sdl2/test/events_test.py'
adding 'sdl2/test/filesystem_test.py'
adding 'sdl2/test/gamecontroller_test.py'
adding 'sdl2/test/guid_test.py'
adding 'sdl2/test/hidapi_test.py'
adding 'sdl2/test/hints_test.py'
adding 'sdl2/test/joystick_test.py'
adding 'sdl2/test/keyboard_test.py'
adding 'sdl2/test/locale_test.py'
adding 'sdl2/test/log_test.py'
adding 'sdl2/test/metal_test.py'
adding 'sdl2/test/mouse_test.py'
adding 'sdl2/test/pixels_test.py'
adding 'sdl2/test/platform_test.py'
adding 'sdl2/test/power_test.py'
adding 'sdl2/test/rect_test.py'
adding 'sdl2/test/render_test.py'
adding 'sdl2/test/rwops_test.py'
adding 'sdl2/test/sdl2ext_algorithms_test.py'
adding 'sdl2/test/sdl2ext_array_test.py'
adding 'sdl2/test/sdl2ext_color_test.py'
adding 'sdl2/test/sdl2ext_displays_test.py'
adding 'sdl2/test/sdl2ext_draw_test.py'
adding 'sdl2/test/sdl2ext_ebs_test.py'
adding 'sdl2/test/sdl2ext_events_test.py'
adding 'sdl2/test/sdl2ext_font_test.py'
adding 'sdl2/test/sdl2ext_image_test.py'
adding 'sdl2/test/sdl2ext_input_test.py'
adding 'sdl2/test/sdl2ext_mouse_test.py'
adding 'sdl2/test/sdl2ext_msgbox_test.py'
adding 'sdl2/test/sdl2ext_particles_test.py'
adding 'sdl2/test/sdl2ext_pixelaccess_test.py'
adding 'sdl2/test/sdl2ext_renderer_test.py'
adding 'sdl2/test/sdl2ext_resources_test.py'
adding 'sdl2/test/sdl2ext_sprite_test.py'
adding 'sdl2/test/sdl2ext_spritesystem_test.py'
adding 'sdl2/test/sdl2ext_surface_test.py'
adding 'sdl2/test/sdl2ext_test.py'
adding 'sdl2/test/sdl2ext_uisystem_test.py'
adding 'sdl2/test/sdl2ext_window_test.py'
adding 'sdl2/test/sdl_test.py'
adding 'sdl2/test/sdlgfx_test.py'
adding 'sdl2/test/sdlimage_test.py'
adding 'sdl2/test/sdlmixer_test.py'
adding 'sdl2/test/sdlttf_test.py'
adding 'sdl2/test/sensor_test.py'
adding 'sdl2/test/shape_test.py'
adding 'sdl2/test/surface_test.py'
adding 'sdl2/test/syswm_test.py'
adding 'sdl2/test/timer_test.py'
adding 'sdl2/test/touch_test.py'
adding 'sdl2/test/version_test.py'
adding 'sdl2/test/video_test.py'
adding 'sdl2/test/vulkan_test.py'
adding 'sdl2/test/resources/animationtest.gif'
adding 'sdl2/test/resources/animationtest.webp'
adding 'sdl2/test/resources/font.bmp'
adding 'sdl2/test/resources/resources.tar.gz'
adding 'sdl2/test/resources/resources.zip'
adding 'sdl2/test/resources/rwopstest.txt'
adding 'sdl2/test/resources/soundtest.mp3'
adding 'sdl2/test/resources/surfacetest.bmp'
adding 'sdl2/test/resources/surfacetest.cur'
adding 'sdl2/test/resources/surfacetest.gif'
adding 'sdl2/test/resources/surfacetest.ico'
adding 'sdl2/test/resources/surfacetest.jpg'
adding 'sdl2/test/resources/surfacetest.lbm'
adding 'sdl2/test/resources/surfacetest.pbm'
adding 'sdl2/test/resources/surfacetest.pcx'
adding 'sdl2/test/resources/surfacetest.pgm'
adding 'sdl2/test/resources/surfacetest.png'
adding 'sdl2/test/resources/surfacetest.pnm'
adding 'sdl2/test/resources/surfacetest.ppm'
adding 'sdl2/test/resources/surfacetest.qoi'
adding 'sdl2/test/resources/surfacetest.svg'
adding 'sdl2/test/resources/surfacetest.tga'
adding 'sdl2/test/resources/surfacetest.tif'
adding 'sdl2/test/resources/surfacetest.webp'
adding 'sdl2/test/resources/surfacetest.xcf'
adding 'sdl2/test/resources/surfacetest.xpm'
adding 'sdl2/test/resources/tuffy.ttf'
adding 'pysdl2-0.9.18a1.dist-info/METADATA'
adding 'pysdl2-0.9.18a1.dist-info/WHEEL'
adding 'pysdl2-0.9.18a1.dist-info/top_level.txt'
adding 'pysdl2-0.9.18a1.dist-info/RECORD'
removing build/bdist.linux-x86_64/wheel
Successfully built pysdl2-0.9.18a1-py3-none-any.whl
Finished creating a wheel...
Finished executing pypaBuildPhase
Running phase: pythonRuntimeDepsCheckHook
Executing pythonRuntimeDepsCheck
Checking runtime dependencies for pysdl2-0.9.18a1-py3-none-any.whl
Finished executing pythonRuntimeDepsCheck
Running phase: installPhase
Executing pypaInstallPhase
Successfully installed pysdl2-0.9.18a1-py3-none-any.whl
Finished executing pypaInstallPhase
Running phase: pythonOutputDistPhase
Executing pythonOutputDistPhase
Finished executing pythonOutputDistPhase
Running phase: fixupPhase
shrinking RPATHs of ELF executables and libraries in /nix/store/n18cc6k6bin4f5kbl6fkisv509yg34c4-python3.13-pysdl2-0.9.17-unstable-2025-04-03
checking for references to /build/ in /nix/store/n18cc6k6bin4f5kbl6fkisv509yg34c4-python3.13-pysdl2-0.9.17-unstable-2025-04-03...
patching script interpreter paths in /nix/store/n18cc6k6bin4f5kbl6fkisv509yg34c4-python3.13-pysdl2-0.9.17-unstable-2025-04-03
stripping (with command strip and flags -S -p) in  /nix/store/n18cc6k6bin4f5kbl6fkisv509yg34c4-python3.13-pysdl2-0.9.17-unstable-2025-04-03/lib
shrinking RPATHs of ELF executables and libraries in /nix/store/nfmkzbqnvrhg8a1aa7342lkkv72s0plb-python3.13-pysdl2-0.9.17-unstable-2025-04-03-dist
checking for references to /build/ in /nix/store/nfmkzbqnvrhg8a1aa7342lkkv72s0plb-python3.13-pysdl2-0.9.17-unstable-2025-04-03-dist...
patching script interpreter paths in /nix/store/nfmkzbqnvrhg8a1aa7342lkkv72s0plb-python3.13-pysdl2-0.9.17-unstable-2025-04-03-dist
Executing pythonRemoveTestsDir
Finished executing pythonRemoveTestsDir
Running phase: installCheckPhase
no Makefile or custom installCheckPhase, doing nothing
Running phase: pythonCatchConflictsPhase
Running phase: pythonRemoveBinBytecodePhase
Running phase: pythonImportsCheckPhase
Executing pythonImportsCheckPhase
Check whether the following modules can be imported: sdl2
Running phase: pytestCheckPhase
Executing pytestCheckPhase
pytest flags: -m pytest -k not\ \(test_SDL_GetPrefPath\)\ and\ not\ \(test_SDL_SetWindowDisplayMode\)\ and\ not\ \(test_SDL_SetWindowFullscreen\)\ and\ not\ \(test_SDL_GetPlatform\)
============================= test session starts ==============================
platform linux -- Python 3.13.9, pytest-8.4.2, pluggy-1.6.0
rootdir: /build/source
configfile: pytest.ini
collected 1101 items / 4 deselected / 1 skipped / 1097 selected                

sdl2/test/audio_test.py .......ssss.........sssssssssssssssssssssss      [  3%]
sdl2/test/blendmode_test.py s                                            [  4%]
sdl2/test/clipboard_test.py s.                                           [  4%]
sdl2/test/cpuinfo_test.py ....................ss                         [  6%]
sdl2/test/endian_test.py .....                                           [  6%]
sdl2/test/error_test.py ..                                               [  6%]
sdl2/test/events_test.py .............................ssssssssssssssss   [ 10%]
sdl2/test/filesystem_test.py .                                           [ 11%]
sdl2/test/gamecontroller_test.py ss.........s....................sss..ss [ 14%]
ssssssssss.....ss                                                        [ 16%]
sdl2/test/guid_test.py ..                                                [ 16%]
sdl2/test/hidapi_test.py ...ssssssssss                                   [ 17%]
sdl2/test/hints_test.py ........                                         [ 18%]
sdl2/test/joystick_test.py .........................................s... [ 22%]
...s.                                                                    [ 22%]
sdl2/test/keyboard_test.py .................                             [ 24%]
sdl2/test/locale_test.py .                                               [ 24%]
sdl2/test/log_test.py ............                                       [ 25%]
sdl2/test/metal_test.py sss                                              [ 25%]
sdl2/test/mouse_test.py sssssssssssssssss                                [ 27%]
sdl2/test/pixels_test.py ..............................                  [ 30%]
sdl2/test/power_test.py .                                                [ 30%]
sdl2/test/rect_test.py .............................................     [ 34%]
sdl2/test/render_test.py ..............s........sssss....sss......ssssss [ 38%]
sssssssssssss..sssssss.                                                  [ 40%]
sdl2/test/rwops_test.py ..s..............................                [ 43%]
sdl2/test/sdl2ext_algorithms_test.py ....                                [ 44%]
sdl2/test/sdl2ext_array_test.py .............                            [ 45%]
sdl2/test/sdl2ext_color_test.py ..............................           [ 47%]
sdl2/test/sdl2ext_displays_test.py ..s...ss                              [ 48%]
sdl2/test/sdl2ext_draw_test.py ...                                       [ 48%]
sdl2/test/sdl2ext_ebs_test.py ................                           [ 50%]
sdl2/test/sdl2ext_events_test.py ........                                [ 51%]
sdl2/test/sdl2ext_font_test.py ......................                    [ 53%]
sdl2/test/sdl2ext_image_test.py ......                                   [ 53%]
sdl2/test/sdl2ext_input_test.py ......                                   [ 54%]
sdl2/test/sdl2ext_mouse_test.py .....s                                   [ 54%]
sdl2/test/sdl2ext_msgbox_test.py ..ss                                    [ 55%]
sdl2/test/sdl2ext_particles_test.py ........                             [ 55%]
sdl2/test/sdl2ext_pixelaccess_test.py ....                               [ 56%]
sdl2/test/sdl2ext_renderer_test.py ...ss........sss                      [ 57%]
sdl2/test/sdl2ext_resources_test.py ...........                          [ 58%]
sdl2/test/sdl2ext_sprite_test.py ............                            [ 59%]
sdl2/test/sdl2ext_spritesystem_test.py .....s....ss...sss                [ 61%]
sdl2/test/sdl2ext_surface_test.py .                                      [ 61%]
sdl2/test/sdl2ext_test.py ....                                           [ 61%]
sdl2/test/sdl2ext_uisystem_test.py ssssssssssssss                        [ 63%]
sdl2/test/sdl2ext_window_test.py ...sss....                              [ 64%]
sdl2/test/sdl_test.py ...                                                [ 64%]
sdl2/test/sdlgfx_test.py ...X.ssssssssssssssssssssssssssssssssss....     [ 68%]
sdl2/test/sdlimage_test.py ........s.....s...........s.................. [ 72%]
s...........                                                             [ 73%]
sdl2/test/sdlmixer_test.py ....s......sss..........sssssssssssssssssssss [ 77%]
sssssssssssssssssssssssssssssss.ssssssssssssss                           [ 81%]
sdl2/test/sdlttf_test.py ..............ss............................... [ 86%]
.......                                                                  [ 86%]
sdl2/test/sensor_test.py .ssssss                                         [ 87%]
sdl2/test/surface_test.py ......s...ss..........F....s..sssss            [ 90%]
sdl2/test/syswm_test.py s                                                [ 90%]
sdl2/test/timer_test.py .....XX                                          [ 91%]
sdl2/test/touch_test.py ......                                           [ 91%]
sdl2/test/version_test.py .......                                        [ 92%]
sdl2/test/video_test.py .................s...s...s...................ss. [ 96%]
.ss.....ssss.s...ss..ssssssss                                            [ 99%]
sdl2/test/vulkan_test.py ssssss                                          [100%]

=================================== FAILURES ===================================
____________________ TestSDLSurface.test_SDL_GetSetClipRect ____________________

self = <sdl2.test.surface_test.TestSDLSurface object at 0x7ffdefa5a140>

    def test_SDL_GetSetClipRect(self):
        rectlist = ((rect.SDL_Rect(0, 0, 0, 0), SDL_FALSE, True),
                    (rect.SDL_Rect(2, 2, 0, 0), SDL_FALSE, False),
                    (rect.SDL_Rect(2, 2, 5, 1), SDL_TRUE, True),
                    (rect.SDL_Rect(6, 5, 10, 3), SDL_TRUE, False),
                    (rect.SDL_Rect(0, 0, 10, 10), SDL_TRUE, True),
                    (rect.SDL_Rect(0, 0, -10, 10), SDL_FALSE, False),
                    (rect.SDL_Rect(0, 0, -10, -10), SDL_FALSE, False),
                    (rect.SDL_Rect(-10, -10, 10, 10), SDL_FALSE, False),
                    (rect.SDL_Rect(10, -10, 10, 10), SDL_FALSE, False),
                    (rect.SDL_Rect(10, 10, 10, 10), SDL_TRUE, False)
                    )
    
        clip = rect.SDL_Rect()
        sf = sdl2.SDL_CreateRGBSurface(0, 15, 15, 32, 0, 0, 0, 0)
        assert isinstance(sf.contents, sdl2.SDL_Surface)
        sdl2.SDL_GetClipRect(sf, byref(clip))
        assert clip == rect.SDL_Rect(0, 0, 15, 15)
    
        for r, clipsetval, cmpval in rectlist:
            retval = sdl2.SDL_SetClipRect(sf, r)
            sdl2.SDL_GetClipRect(sf, byref(clip))
            err = "Could not set clip rect %s" % r
            assert retval == clipsetval, "retval: " + err
>           assert (clip == r) == cmpval, "clip: " + err
E           AssertionError: clip: Could not set clip rect SDL_Rect(x=2, y=2, w=0, h=0)
E           assert (SDL_Rect(x=2, y=2, w=0, h=0) == SDL_Rect(x=2, y=2, w=0, h=0)
E             
E             Use -v to get more diff) == False

sdl2/test/surface_test.py:493: AssertionError
=========================== short test summary info ============================
FAILED sdl2/test/surface_test.py::TestSDLSurface::test_SDL_GetSetClipRect - AssertionError: clip: Could not set clip rect SDL_Rect(x=2, y=2, w=0, h=0)
===== 1 failed, 770 passed, 324 skipped, 4 deselected, 3 xpassed in 13.47s =====
```

### Additional context

_No response_

### System metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.12.57, NixOS, 25.11 (Xantusia), 25.11pre-git`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.31.2`
 - nixpkgs: `/nix/store/cjjpqrp9xy44j5fjh3mygyxy3i7mslj2-source`

### Notify maintainers

@pmiddend 

---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/development/python-modules/pysdl2/default.nix
+++ b/pkgs/development/python-modules/pysdl2/default.nix
@@ -82,10 +82,9 @@ buildPythonPackage rec {
     # GetPrefPath for OrgName/AppName is None
     "test_SDL_GetPrefPath"
 
-    # TODO: Remove once sdl2-compat is updated to 2.32.56
-    "test_SDL_SetWindowDisplayMode"
-    "test_SDL_SetWindowFullscreen"
-    "test_SDL_GetPlatform"
+    # AssertionError:
+    # clip: Could not set clip rect SDL_Rect(x=2, y=2, w=0, h=0)
+    "test_SDL_GetSetClipRect"
   ];
 
   meta = {

